Reorg part1 redirect tutorials (#793)
solid/solidproject.org: _posts/developers/apps/vocabularies/2019-01-01-11_creation-good-practices.md

There are different "usual" ways to link to other vocabularies:

- **Equivalence**: You state that your term and the other represent exactly the same concept, and that logically, anything that goes for one goes for the other. For instance, we could say that `webco:Webcomic owl:sameAs schema:CreativeWork`. It is quite a strong statement, but it makes it possible to make plenty of deductions using a reasoner. The downside is that if you add a property to your local term, that is therefore considered applied to the other term, which can lead to contradiction when your vocabulary is reused. Let's imagine that the Webcomic vocabulary contains the triple `webco:Webcomic owl:disjointFrom ex:Book` (where `ex:Book` is a well-known term for books). If a book ontology importing our webcomic vocabulary states that `ex:Book rdfs:subClassOf schema:CreativeWork`, an inconsistency is created, because `ex:Book` is both a subclass of and disjoint from `schema:CreativeWork`. The take-away message here is that equivalence should be used wisely.

> - **Subclassing**: You state that your concept is a subclass of the remote concept you want to extend. It is not as powerful as equivalence, but it also causes less trouble. In this case, we would say that `webco:Webcomic rdfs:subClassOf schema:CreativeWork`.

Reviewer: ```suggestion
- **Subclassing**: You state that your concept is a subclass of the remote concept you want to extend. It is not as powerful as equivalence, but it also has less potential to cause trouble. In this case, we would say that `webco:Webcomic rdfs:subClassOf schema:CreativeWork`.
```

> - **Direct reuse**: In this case, you do not create a new term at all in your vocabulary, and you use the remote term directly. For instance, we could say that `webco:follows rdfs:domain schema:Person` instead of `webco:follows rdfs:domain webco:Person` and `webco:Person rdfs:subClassOf schema:Person`. This prevents redefining existing terms, but if not done carefully, it can create the same issues as equivalence.

Reviewer: ```suggestion
- **Direct reuse**: In this case, you do not create a new term in your vocabulary at all; instead, you use the remote term directly. For instance, we could say that `webco:follows rdfs:domain schema:Person .` instead of `webco:follows rdfs:domain webco:Person . webco:Person rdfs:subClassOf schema:Person .`. This prevents redefining such existing terms, but if not done carefully, it can create issues similar to those created by incorrect equivalence.
```
